cmd/queue/main.go

```diff
@@ -91,6 +95,12 @@ func initEnv() {
 
 	// TODO(mattmoor): Move this key to be in terms of the KPA.
 	servingRevisionKey = autoscaler.NewKpaKey(servingNamespace, servingRevision)
+	health = &healthServer{alive: true}
```

Comment:
Any reason to move this here? Seems leftover from refactoring?

Reply:
It did not make sense to keep it where it was based on the flow of the code. With the exception of the global channels a lof of the globals were initialized in `initEnv`...